src/main/kotlin/com/lykke/trade/volume/monitoring/service/spring/ApplicationConfig.kt

```diff
@@ -0,0 +1,37 @@
+package com.lykke.trade.volume.monitoring.service.spring
+
+import com.lykke.trade.volume.monitoring.service.config.TradeVolumeConfig
+import com.lykke.utils.config.ConfigInitializer
+import org.slf4j.LoggerFactory
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.core.env.Environment
+import org.springframework.core.env.Profiles
+
+@Configuration
+open class ApplicationConfig {
+
+    companion object {
+        private val LOGGER = LoggerFactory.getLogger(ApplicationConfig::class.java.name)
+    }
+
+    @Bean
+    open fun config(environment: Environment): TradeVolumeConfig? {
```

Comment:
Here should be returned Config-wrapper with `TradeVolumeConfig` field as we load config from global json with many sections

Reply:
fixed